signalfx/tracing.go

```diff
@@ -185,25 +187,21 @@ func (th *traceSpanHandler) convertInstance(istioSpan *tracespan.InstanceMsg) *t
 	tags := map[string]string{}
 
 	// Special handling for the span name since the Istio attribute that is the
-	// most suited for the span name is request.path which can contain query
-	// params which could cause high cardinality of the span name.
-	spanName := istioSpan.SpanName
-	if strings.Contains(spanName, "?") {
-		idx := strings.LastIndex(spanName, "?")
-		qs := spanName[idx+1:]
-		if vals, err := url.ParseQuery(qs); err == nil {
-			for k, v := range vals {
-				tags[k] = strings.Join(v, "; ")
-			}
-		}
-
-		spanName = spanName[:idx]
-	}
+	// most suited for the span name is request.path which  could cause high
+	// cardinality of the span name.
+	tags[tagIstioSpanKey] = istioSpan.SpanName
```

Comment:
I would also try to handle this in https://github.com/signalfx/signalfx-istio-adapter/blob/master/resources/tracespan.yaml#L14 to give users more flexibility to change it if they want.